Mathlib/Algebra/SMulWithZero.lean

```diff
@@ -175,7 +175,13 @@ protected lemma MulActionWithZero.nontrivial
 #align mul_action_with_zero.nontrivial MulActionWithZero.nontrivial
 
 variable {R M}
-variable [MulActionWithZero R M] [Zero M'] [SMul R M']
+variable [MulActionWithZero R M] [Zero M'] [SMul R M'] (p : Prop) [Decidable p]
```

Comment:
nit: should `P` be upper case?

Reply:
Depends which file you ask! I would say no here just because `R` is a close letter used for an algebraic structure.